mappings/net/minecraft/item/WrittenBookItem.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/class_1843 net/minecraft/item/WrittenBookItem
+	FIELD field_30929 MAX_TITLE_SELECTION_LENGTH I
 	FIELD field_30930 MAX_TITLE_LENGTH I
+	FIELD field_30931 MAX_PAGE_SELECTION_LENGTH I
+	FIELD field_30932 MAX_PAGES_LENGTH I
```

Comment:
`MAX_PAGE_LENGTH`?

Reply:
It's the max length of the `Pages` nbt key, hence the name

Reply:
No. It's the max length of an element in the `Pages` nbt key. `getString` is an index access.

![image](https://user-images.githubusercontent.com/69905075/195031728-f2010466-f679-42b7-a5b4-4818b70228fe.png)

Reply:
Totally misread the code, changed in latest commit